Lost heartbeat schedule notification + Fixes & improvements

Fralez opened the pull request:
- Schedule notifications done
- LocalNotificationIds system created
- Re-random daily biometric notification improved
- Removed mistakenly used doc comments

Fralez committed:
Feature finished and some fixes added:
- Schedule notifications done
- LocalNotificationIds system created
- Re-random daily biometric notification improved
- Remvoed mistakenly used doc comments

Fralez commented on lib/services/LocalNotificationService.dart:
```diff
@@ -9,6 +9,18 @@ AndroidInitializationSettings initializationSettingsAndroid;
+  lostHeartbeatScheduled,
+  lostConnectivityInstant,
+  lostGpsInstant,
+  testSchedule
```
Forgot to delete this, I will be updating ASAP

Fralez committed:
Removed mock localnotification id

cherta commented on lib/components/pages/home_page.dart:
```diff
@@ -67,8 +67,10 @@ class _HomePageState extends State<HomePage> {
-            .navigateTo('alertpagebiometrics', payload.split("-")[1]);
+        locator<NavigationService>().navigateTo(
+            'alertpagebiometrics',
+            LocalNotificationIds.values
```
Same way you have an enum I think you can provide a method that abstracts all this ugliness (?), something similar to `LocalNotificationIds.getId(payload)`

cherta commented on lib/services/BiometricAuthService.dart:
```diff
@@ -17,28 +20,54 @@ Future<bool> authenticateFingerprint(int notificationId) async {
+/// If [id] == `middayDaily` -> Re-random `LocalNotificationIds.afternoonDaily`
+///
+/// If [id] == `afternoonDaily` -> Re-random `LocalNotificationIds.morningDaily`
+void reRandomDailyNotifications(notificationId) {
```
I'll change the method name to something less cryptic like `reOrderDailyNotifications`

cherta reviewed (approved):
The code is well done, I still haven't wrapped my head around the `generateScheduledNotification` in `lib/services/GeofencingService.dart`, maybe we can sync offline about those changes, I would like to understand the intention behind the code.

Fralez committed:
Stylish changes:
- Renamed reOrderDailyNotifications method
- Wrapped up in a function notification id from payload parser
